From: Borislav Petkov <bp@suse.de>
Subject: xen/x86/kaiser: Rename and simplify X86_FEATURE_KAISER handling
Patch-mainline: Never, SUSE-Xen specific

Concentrate it in arch/x86/mm/kaiser.c and use the upstream string "nopti".

Signed-off-by: Borislav Petkov <bp@suse.de>
Automatically created from "patches.kaiser/kaiser-nokaiser-0007-rename-and-simplify-feature-setting.patch" by xen-port-patches.py

--- a/arch/x86/kernel/cpu/common-xen.c
+++ b/arch/x86/kernel/cpu/common-xen.c
@@ -698,10 +698,6 @@ void __cpuinit get_cpu_cap(struct cpuinf
 #endif
 
 	init_scattered_cpuid_features(c);
-#ifdef CONFIG_KAISER
-	if (kaiser_enabled)
-		set_cpu_cap(c, X86_FEATURE_KAISER);
-#endif
 }
 
 static void __cpuinit identify_cpu_without_cpuid(struct cpuinfo_x86 *c)
